lua/gitblame/init.lua

```diff
@@ -639,7 +639,7 @@ M.disable = function(force)
     if not vim.g.gitblame_enabled and not force then
         return
     end
-
+    current_blame_text = ""
```

Comment:
Call this after everything else (after `last_position`) and set it to `nil` instead of an empty string